qiskit/circuit/instruction.py

```diff
@@ -325,9 +325,14 @@ def inverse(self):
                                        params=self.params.copy())
 
         else:
+            if getattr(self, 'label', None) is None:
```

Comment:
`if self.label is None:` ?

Reply:
Hi @itoko, I tried that before. There are cases where the 'self' calling the inverse is of `Instruction` class, which does not have any `label` attribute. Thus attempting to read `self.label` will throw an error. Instead, by using `getattr` we are checking whether `self` has the attribute `label` before trying to read it.